### DISCUSSION TITLE
Missing/Wrong/Broken Recipes Summary

### DISCUSSION THREAD

**Discussion by DevDyna** [DevDyna/IronBerry#27] [Common Bugs]
Write a comment of any missing/wrong recipe that you can find 

- [x] cutting board
- [x] sticks (planks)
- [x] bark knife
- [x] Old Industrial Wood Planks 
- [x] Inventory reader
___
Other:
- [x] Rework fishing
- [x] Apple Sprouts shenanigan
- [x] Pressure Chamber dont accept some item
- [x] Configurator recipe cause bug on alloyfurnace

**Comment by aswan1239** [comment]
I might have overlooked it, but wooden bowl is missing, I believe.

**Reply by DevDyna** [reply]
> I might have overlooked it, but wooden bowl is missing, I believe.

did you have updated ironberry on 0.9c hotfix-2 ?

**Reply by aswan1239** [reply]
I only had 0.9c hotfix. I ran into the problem this morning but just posted now. I'll update and take a look.

**Reply by aswan1239** [reply]
Cutting board?

**Reply by DevDyna** [reply]
> Cutting board?

fixed as completed

**Comment by ZariusCorten** [comment]
Several Aquatic Entangler added recipes are non-functioning. (They appear in the JEI interface, but don't actually drop.)

From Junk Net:
Ghast Tear
Soul Jar
Frogglin Egg
Feather

I only ran one stack each of the Aqua Chow and Benthic Aqua-Chow, so I'm not 100% certain about it, but I also didn't pull any Phantom Membranes, Rabbit's Foot, Prismarine Shards, Meat, Bamboo, Kelp, or Iron Coins (from the machine, the quest drops are working fine).

**Reply by DevDyna** [reply]
fixed as reverted to mono-loot table so it *should* allow to generate all items without need to make multiple fishers but disabled fish foods due buggy recipes

**Comment by ZariusCorten** [comment]
Apple Sprouts aren't working, either. Craftable, placeable, grow to maturity. Can't harvest them. I haven't gotten (back) to the point of diamonds yet, but both regular and Emerald apple sprouts don't harvest.

**Reply by DevDyna** [reply]
fixed as impossible -> sprouts are now disabled (mod issue)

**Comment by ZariusCorten** [comment]
The Pressure Chamber recipe for the various material baring berries doesn't work, due to the Interface not accepting them in to the machine.

**Reply by DevDyna** [reply]
> The Pressure Chamber recipe

maybe did you intend when input can make multiple output ? 
Due i have try to reproduce but berries recipe dont cause any issue

example
`teslatite dust + 2x redstone torch -> 6x capacitor`
but also can make
`teslatite dust -> 16x gravedust`
so it craft only the gravedust

**Comment by ZariusCorten** [comment]
Configurator recipe doesn't work. The alloy furnace just turns the Battery Block in to 3 units of iron.

**Reply by DevDyna** [reply]
fixed -> replaced alloy furnace recipe with shapeless recipe

**Comment by ZariusCorten** [comment]
Inventory reader has no recipe.

**Reply by DevDyna** [reply]
fixed as completed

**Comment by ZariusCorten** [comment]
Magic Lollipop recipe is missing, making the quest of the same name incompleteable if you ate all of the starter ones you get. 

All forms of aquatic entangler modifying items are missing their recipes, also rendering the quest of the same name incompleteable.

I'm also going to include this here because I don't know where else to put it: The "A Furious Cocktail" quest doesn't give any information about exactly *what* A Furious Cocktail is, other than it clearly being one or more potions mixed together.

**Reply by DevDyna** [reply]
> Magic Lollipop recipe is missing, making the quest of the same name incompleteable if you ate all of the starter ones you get.

right , it was caused by a corail tombstone changes that i have forgot to see

 
> All forms of aquatic entangler modifying items are missing their recipes, also rendering the quest of the same name incompleteable.

i completly forgot to remove that from the quest...
(due they dont work with custom loottables and some times can stuck the entire production)

> I'm also going to include this here because I don't know where else to put it: The "A Furious Cocktail" quest doesn't give any information about exactly what A Furious Cocktail is, other than it clearly being one or more potions mixed together.

modified to make it more intuitive (also less *extreme* as quest due too many effects)

**Reply by ZariusCorten** [reply]
I appreciate the responses! Many thanks!

**Comment by ZariusCorten** [comment]
Also, I hate that the items in the quest log can't be activated with the A or R buttons. :P

**Reply by DevDyna** [reply]
> Also, I hate that the items in the quest log can't be activated with the A or R buttons. :P

yeah... many mods add qol features but none of that want to backport that but only want work on +1.21.x that still completly unstable...

**Reply by ZariusCorten** [reply]
Ah well, it's a minor convenience. If it doesn't come back around, it doesn't come back around. Won't kill my fun.

**Comment by ZariusCorten** [comment]
Just noticed that Zinc doesn't pop up in the JEI (The Blue Power one for batteries). Like... At all. It shows up in the Blue Power battery recipe, but it doesn't show up in the search, nor does it bring up a recipe for it.

Though, unless you're intentionally using it as a gate, you might consider just replacing it with the Create Zinc, just to clean up the "multiple items of the same name" problem you run in to with tech mods that focus on metallurgy.

**Reply by DevDyna** [reply]
> Just noticed that Zinc doesn't pop up in the JEI (The Blue Power one for batteries). Like... At all. It shows up in the Blue Power battery recipe, but it doesn't show up in the search, nor does it bring up a recipe for it.
>
> Though, unless you're intentionally using it as a gate, you might consider just replacing it with the Create Zinc, just to clean up the "multiple items of the same name" problem you run in to with tech mods that focus on metallurgy.

Idk how that recipe was ignored by filters... 
(i have completly removed that item so that was why on jei dont show and dont have a recipe)

fixed as completed on 0.9e-hotfix1

**Comment by ZariusCorten** [comment]
To start with, this is _non_-critical, in that it doesn't need a hotfix, just addressed for the next main update.

Strange Christmas hat doesn't show up in the JEI interface when searching. It shows up just fine if you look up the uses for the merchant stone, so you can still find the recipe, and it still works just fine. It's just not showing up in the search.

**Reply by DevDyna** [reply]
> Strange Christmas hat doesn't show up in the JEI interface when searching [...]

it probably was caused by recent corail tombstone update or  a tombstone *game event* that hide/show festive items on jei so nothing of "too big" -> i will fix that on next release / hotfix

**Comment by ZariusCorten** [comment]
Gold weapons/armor don't get cooked in the Alloy Furnace or the Blulectric Alloy Furnace. The recipes for them show up just fine, it's just not actually working. Used to work.

**Reply by ZariusCorten** [reply]
I'm just making due with tossing them off the edge of my base and using the Give command, so don't feel like it's critical just for me. :P

**Reply by DevDyna** [reply]
> Gold weapons/armor don't get cooked in the Alloy Furnace or the Blulectric Alloy Furnace. The recipes for them show up just fine, it's just not actually working. Used to work.

i can't do nothing about that (it was related to "priority recipe") -> all recycle recipes are disabled but bluepower keep it on jei (bluepower-issue)

**Reply by ZariusCorten** [reply]
Well, that's a shame. Maybe adjust the furnace recipe to reflect what the alloy furnace would have given? Just a thought.

**Comment by ZariusCorten** [comment]
Not so much a missing/wrong/broken recipe as an issue with a machine *making* that recipe... The recipe to make Radioactive Uranium/Thorium requires 10,000 mb of Enriched Osmium, but the Fluid Encapsulator only has an input of 8,000 mb, and the machine upgrades - including the one that would up tank size - have their recipes disabled.

**Reply by DevDyna** [reply]
> Not so much a missing/wrong/broken recipe as an issue with a machine making that recipe... The recipe to make Radioactive Uranium/Thorium requires 10,000 mb of Enriched Osmium, but the Fluid Encapsulator only has an input of 8,000 mb, and the machine upgrades - including the one that would up tank size - have their recipes disabled.

atm all thermal machines are experimental (and i forgot to commit last commit...)
https://github.com/DevDyna/IronBerry/commit/3cabe106fe616781b88f198e7ec21bfe25369fc4

**Reply by ZariusCorten** [reply]
Well, all of the one's I've tested so far seem to be working perfectly.

**Reply by ZariusCorten** [reply]
I stand corrected, there is a way to upgrade the Fluid Encapsulator's tank. The "Integral Components" items increase both machine speed and tank size. I thought they only multiplied the additions from other upgrades.

**Comment by ZariusCorten** [comment]
Oh! A couple of notes I wanted to make. I don't know that there is anything you can do about them, or if they're really even 'problems', but:

1) NuclearCraft Energy Upgrades don't do anything. This isn't a huge deal as the speed upgrades are only additive in how much energy they take, but I thought it was worth a mention.

2) None of the Source Stone secondary options are selectable with the Chisel. While mildly inconvenient, it's not enough so to stop me from using it for most of my base, once I got access to it.

**Reply by ZariusCorten** [reply]
A couple of random thoughts I had:

Add a recipe to the Phytogenic Insolator to convert the Organic Compost block in to Rich Soil?

Add a recipe to the Fluid Encapsulator to convert the Dry Peat Bog in to Saturated Peat Bog?

**Reply by DevDyna** [reply]
> NuclearCraft Energy Upgrades don't do anything. This isn't a huge deal as the speed upgrades are only additive in how much energy they take, but I thought it was worth a mention.

ik , NCN fix that (not entirely) but never fix kubejs incompatibility that persist so... if i update it , the entire modpack *blow up*


> None of the Source Stone secondary options are selectable with the Chisel. While mildly inconvenient, it's not enough so to stop me from using it for most of my base, once I got access to it.

👍 

> Add a recipe to the Phytogenic Insolator to convert the Organic Compost block in to Rich Soil?
 Add a recipe to the Fluid Encapsulator to convert the Dry Peat Bog in to Saturated Peat Bog?

not a bad idea , i will add

**Reply by ZariusCorten** [reply]
If fixing the problem blows up the mod, leave it as is. Like I said, it's not really a problem for game play, just something I noticed while I was futzing around with stuff.

**Reply by ZariusCorten** [reply]
Another thought I had, for the sake of conformity, was maybe you could put the Dirt -> Farmland and Farmland -> Podzal recipes in the Pulverizer? The Methane chain never stops being useful, and if most of the chain is going to be present, seems like we might as well just make the whole chain available from start to finish. 

Edit: Or, rather, the most efficient recipe for it. Using Dirt is easy enough, it just feels like we're just going "Oh, that's good enough, we can just leave it like this."

**Reply by DevDyna** [reply]
> Dirt -> Farmland and Farmland -> Podzal recipes in the Pulverizer

ehm... it wasn't a "feature" but a bug...
so... i will replace dirt with grass block (with more output)

**Reply by ZariusCorten** [reply]
Hah. Well, at least I found a bug, I suppose. Thank you for addressing it!

**Comment by ZariusCorten** [comment]
Okay, I would absolutely *love* a way to convert Iron Coins in to Electum Coins without being required to use the Amadron Tablet. It's great for buying things, but converting literally hundreds of thousands coins is... Unpleasantly tedious. An Induction Furnace recipe, perhaps?

**Reply by DevDyna** [reply]
i prefered add it to the extended compressor due it can be used as "uncraft"

**Reply by ZariusCorten** [reply]
As long as  can automate it, I'll take what I can get. Thanks!

**Comment by ZariusCorten** [comment]
Given that it's used in a pretty late (so far, at least) game recipe, I feel like the Fine Copper Wire from Blue Power needs an automated recipe.

Suggestion: Use the Magma Crucible to make the liquid Copper from Nuclearcraft, then use the Blast Chiller with the Drawplate installed in the Cast socket to make the wire.

**Reply by DevDyna** [reply]
it could be a good recipe but it will require Thermal age so it's too later
https://github.com/DevDyna/IronBerry/commit/a1f8ae0b00d62d9f8614450649609bc0450ea79a

**Reply by ZariusCorten** [reply]
I mean, that's really when the recipe becomes useful. The main use I ran in to for it is the little motors, which I didn't really use until i started trying to make the Machine Frames from thermal. If you want to add one to the Nuclearcraft equivalents, I certainly won't complain. But I also replaced every NC machine I could the second I got thermal foundation, so I suppose that's not a huge priority _to me._ I just thought I'd bring it up, though, and the Blast Chiller is really the first machine I could think of that made sense.

**Reply by DevDyna** [reply]
> I mean, that's really when the recipe becomes useful. The main use I ran in to for it is the little motors, which I didn't really use until i started trying to make the Machine Frames from thermal. If you want to add one to the Nuclearcraft equivalents, I certainly won't complain. But I also replaced every NC machine I could the second I got thermal foundation, so I suppose that's not a huge priority _to me._ I just thought I'd bring it up, though, and the Blast Chiller is really the first machine I could think of that made sense.

probably i didn't explain too much due it was very "back-end stuff" but many NC recipes are directly linked to Thermal recipes so if i add a recipe like that , it will created another recipe on thermal machines
[file](https://github.com/DevDyna/IronBerry/blob/main/kubejs/server_scripts/recipes/nc/manufactory.js#L4)

**Reply by ZariusCorten** [reply]
Ah, that's very interesting. Either way, it was just a(nother) suggestion. Thanks for your time!
